api_tests/e2e/rfc003/btc_eth/restart.ts

```diff
@@ -1,11 +1,12 @@
 import { twoActorTest } from "../../../lib_sdk/actor_test";
+import { AssetKind } from "../../../lib_sdk/asset";
 
 setTimeout(function() {
-    twoActorTest("restart-cnd-after-swap-is-accepted", async function({
+    twoActorTest("rfc003-btc-eth-cnd-can-be-restarted", async function({
```

Comment:
I put this one separate because it only actually tests the database and non of the other logic outside of the database module - that is not at all obvious though.